appveyor.yml

```diff
@@ -49,10 +49,11 @@ install:
     - IF %PHP%==1 echo extension=php_pdo_sqlite.dll >> php.ini-max
     - IF %PHP%==1 echo extension=php_curl.dll >> php.ini-max
     - IF %PHP%==1 echo curl.cainfo=c:\php\cacert.pem >> php.ini-max
-    - appveyor DownloadFile https://getcomposer.org/composer.phar
+    - appveyor DownloadFile https://getcomposer.org/download/1.0.0/composer.phar
     - copy /Y php.ini-max php.ini
+    - composer self-update
+    - composer --version
     - cd c:\projects\symfony
```

Comment:
I suggest moving composer fetching after this line so that we won't touch the cached dir and maybe trigger invalidation.